client/components/UserTile.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+const UserTile = (props) => {
+    const { _id, isAdmin, email } = props.user;
+
+    return (
+        <div className="tile">
+            <div className="tile-content">
+                <p className="tile-title">{email}</p>
+            </div>
+            {isAdmin && 
+                <span className="chip mt-1 mx-1">Admin</span>
+            }
+            <div className="tile-action mx-1">
+                <button 
+                    className="btn btn-primary"
+                    onClick={() => props.onUpdateUser(props.user)}
+                >
+                    Edit
+                </button>
+            </div>
+            <div className="tile-action mx-1">
+                <button 
+                    className="btn btn-error"
+                    onClick={() => props.onDeleteUser(props.user)}
+                >
+                    Remove
+                </button>
+            </div>
+        </div>
+    );
+}
+
+export default UserTile;
```

Comment:
Simple presentational component for rendering each user

Reply:
red at end of file suggests an extra EOL is good

do you know why this is the standard?